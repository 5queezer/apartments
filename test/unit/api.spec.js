import api from '@/assets/api.faker'

describe('Api', () => {
  it('fetches the data', async () => {
    // act
    const result = await api.getApartments()

    console.log(result)
    // assert
    expect(result.length).toEqual(10)
  })
})
