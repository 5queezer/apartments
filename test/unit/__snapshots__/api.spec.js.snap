// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Api matches the snapshot 1`] = `
Array [
  Object {
    "address": Object {
      "city": "Stokesburgh",
      "lat": "-25.5929",
      "long": "-175.3533",
      "street": "Bettye Dam",
      "zip": "21889",
    },
    "bathrooms": 4,
    "bedrooms": 4,
    "id": 0,
    "picture": "http://lorempixel.com/300/200/city?77135",
    "price": 418000,
    "sqm": 148,
    "title": "Distributed hybrid middleware",
  },
  Object {
    "address": Object {
      "city": "Anabelport",
      "lat": "20.7713",
      "long": "140.1214",
      "street": "Rippin Forge",
      "zip": "00728",
    },
    "bathrooms": 2,
    "bedrooms": 5,
    "id": 1,
    "picture": "http://lorempixel.com/300/200/city?9825",
    "price": 199000,
    "sqm": 223,
    "title": "Object-based human-resource open architecture",
  },
  Object {
    "address": Object {
      "city": "South Pasquale",
      "lat": "36.7670",
      "long": "-13.4724",
      "street": "Lockman Camp",
      "zip": "34548-5498",
    },
    "bathrooms": 2,
    "bedrooms": 4,
    "id": 2,
    "picture": "http://lorempixel.com/300/200/city?45272",
    "price": 102000,
    "sqm": 282,
    "title": "Centralized zero defect framework",
  },
  Object {
    "address": Object {
      "city": "Beahanton",
      "lat": "11.0580",
      "long": "-25.1455",
      "street": "Gibson Keys",
      "zip": "16096-9595",
    },
    "bathrooms": 3,
    "bedrooms": 5,
    "id": 3,
    "picture": "http://lorempixel.com/300/200/city?64655",
    "price": 197000,
    "sqm": 280,
    "title": "Reactive exuding conglomeration",
  },
  Object {
    "address": Object {
      "city": "Caseytown",
      "lat": "82.5844",
      "long": "-137.3205",
      "street": "Geovanni Circle",
      "zip": "63947",
    },
    "bathrooms": 1,
    "bedrooms": 4,
    "id": 4,
    "picture": "http://lorempixel.com/300/200/city?70454",
    "price": 320000,
    "sqm": 60,
    "title": "Function-based real-time installation",
  },
  Object {
    "address": Object {
      "city": "Port Georgiannamouth",
      "lat": "69.5270",
      "long": "85.8683",
      "street": "Blaise Coves",
      "zip": "64292",
    },
    "bathrooms": 2,
    "bedrooms": 3,
    "id": 5,
    "picture": "http://lorempixel.com/300/200/city?2581",
    "price": 394000,
    "sqm": 199,
    "title": "Secured tertiary emulation",
  },
  Object {
    "address": Object {
      "city": "Port Morganbury",
      "lat": "22.6326",
      "long": "-119.2834",
      "street": "Justus Passage",
      "zip": "56093-6496",
    },
    "bathrooms": 1,
    "bedrooms": 1,
    "id": 6,
    "picture": "http://lorempixel.com/300/200/city?22921",
    "price": 262000,
    "sqm": 86,
    "title": "Progressive maximized flexibility",
  },
  Object {
    "address": Object {
      "city": "Shawnview",
      "lat": "-3.5353",
      "long": "22.8819",
      "street": "Albin Lake",
      "zip": "11515",
    },
    "bathrooms": 4,
    "bedrooms": 5,
    "id": 7,
    "picture": "http://lorempixel.com/300/200/city?47654",
    "price": 473000,
    "sqm": 232,
    "title": "Reactive clear-thinking capability",
  },
  Object {
    "address": Object {
      "city": "Hildegardmouth",
      "lat": "16.9008",
      "long": "-88.5306",
      "street": "Becker Skyway",
      "zip": "70795-1991",
    },
    "bathrooms": 1,
    "bedrooms": 2,
    "id": 8,
    "picture": "http://lorempixel.com/300/200/city?17755",
    "price": 170000,
    "sqm": 252,
    "title": "Object-based reciprocal internet solution",
  },
  Object {
    "address": Object {
      "city": "West Thaddeus",
      "lat": "40.7985",
      "long": "87.4245",
      "street": "Rhoda Crest",
      "zip": "86263-5179",
    },
    "bathrooms": 1,
    "bedrooms": 1,
    "id": 9,
    "picture": "http://lorempixel.com/300/200/city?69077",
    "price": 428000,
    "sqm": 118,
    "title": "Cross-group upward-trending throughput",
  },
]
`;
